### PR TITLE
virsh_net_update: Add more network section to net_update command tests.

### PR DESCRIPTION
**Pull request by rbian**

### CODE REVIEW

**Comment by yumingfei** [comment]
Test looks good:
```
TOTAL TIME: 522.62 s (08:42)
TESTS PASSED: 56
TESTS FAILED: 0
TESTS SKIPPED: 56
SUCCESS RATE: 100.00 %
```
But I saw many SKIPs like this:
```
10:52:07 INFO | SKIP type_specific.io-github-autotest-libvirt.virsh.net_update.modify.ip_dhcp_range.net_inactive.options_config -> TestNAError: Skip the test: error: Failed to update network updatenet
error: this function is not supported by the connection driver: dhcp ranges cannot be modified, only added or deleted
10:52:07 INFO | 
```
I'm not sure in which environment it would work, or you can try to add more information about them?

File: libvirt/tests/src/virsh_cmd/network/virsh_net_update.py [hunk 1]
```diff
@@ -23,15 +26,41 @@ def run(test, params, env):
...
+<portgroup name='engineering' default='no'>
+  <virtualport type='802.1Qbh'>
+    <parameters profileid='test'/>
+  </virtualport>
+  <bandwidth>
+    <inbound average='1000' peak='5000' burst='5120'/>
+    <outbound average='1000' peak='5000' burst='5120'/>
+  </bandwidth>
+</portgroup>
+"""
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_update.py]: This XML class already defined in ```virttest/libvirt_xml/network_xml.py```

**Comment by will-Do** [comment]
For the SKIP cases, as we discussed, should be Error/Fail I guess

File: libvirt/tests/src/virsh_cmd/network/virsh_net_update.py (hunk 1 above)
Comment (rbian) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_update.py]: I just follow the original code style, it's a convenient way to implement the test. Yes, we can use the class defination.

File: libvirt/tests/src/virsh_cmd/network/virsh_net_update.py [hunk 2]
```diff
@@ -23,15 +26,41 @@ def run(test, params, env):
...
+            del test_xml.mac
+            del test_xml.ip
+        test_xml.define()
+        if net_state == "active":
+            test_xml.start()
+        test_xml.debug_xml()
+        virsh.net_dumpxml(net_name)
     except xcepts.LibvirtXMLError, detail:
         raise error.TestNAError("Failed to define a test network.\n"
                                 "Detail: %s." % detail)
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/network/virsh_net_update.py]: Maybe you can fix this in this patch: s/TestNAError/TestError

**Comment by rbian** [comment]
@will-Do , Skip tests are all not supported:
SKIP type_specific.io-github-autotest-libvirt.virsh.net_update.modify.bridge.net_active.options_no -> TestNAError: Skip the test: error: Failed to update network updatenet
error: this function is not supported by the connection driver: can't update 'bridge' section of network 'updatenet'

**Comment by will-Do** [comment]
Then it's fine for me. As this PR exists for a long time, we merge with it ASAP. @waynesun09 @Hao-Liu

**Comment by waynesun09** [comment]
the test is good, ACK and merging now. 
Thx @rbian @will-Do @yumingfei

**Pull request merged**